src/testing/jest/test/tsconfig.json

```diff
@@ -1,3 +0,0 @@
-{
-  "extends": "../../tsconfig.internal.json"
```

Comment:
This was `git mv`'ed, but then when I updated the value for 'extends' here, the GH diff algo thinks they're entirely different files